Makefile

```diff
@@ -11,7 +11,7 @@ include $(addprefix ./vendor/github.com/openshift/build-machinery-go/make/, \
 	targets/openshift/crd-schema-gen.mk \
 )
 
-EXCLUDE_DIRS := _output/ dependencymagnet/ hack/ third_party/ tls/ tools/ vendor/
+EXCLUDE_DIRS := _output/ dependencymagnet/ hack/ third_party/ tls/ vendor/
```

Comment:
Any idea on why we're no longer excluding the tools directory?

Reply:
This looks like an artifact so a rebase I did recently. Fixing it now.